src/lib/components/Canvas.svelte

```diff
@@ -0,0 +1,157 @@
+<script context="module" lang="ts">
+  import RenderManager from '../util/renderManager';
+  import { getContext as getCTX } from 'svelte';
```

Comment:
There's no need to alias this import, is there?

Reply:
Oddly enough (I'm not sure if this is a bug but I doubt it is) svelte doesn't allow for you to declare the same function in both `context="module"` and the component-attatched script, so the alias is necessary to not conflict with lines 46-48:

```ts
function getContext() {
  return context;
}
```

Reply:
Oh yeah you're totally right